app/design/frontend/base/default/template/oauth2/device/verify.phtml

```diff
@@ -0,0 +1,6 @@
+<form action="<?php echo $this->getFormActionUrl(); ?>" method="post">
+    <?php echo $this->getBlockHtml('formkey') ?>
+    <label for="user_code"><?php echo $this->__('User Code'); ?></label>
```

Comment:
"User Code" should be added to the translations in the Mage_Oauth.csv file.

"Authorize" exists already but please double check.